### PR TITLE
Add team best feature

### PR DESCRIPTION
**Pull request by rohittanwar1**

### CODE REVIEW

File: functions/src/index.ts [hunk 1]
```diff
@@ -115,30 +110,36 @@ async function teamAvg() {
...
+/**
+ * @return {String}
+ **/
+async function teamBestScore() {
+  const teamAverage = (
+    await admin.firestore().collection("TeamAvg").doc("Latest").get()
+  ).data();
+
+  const currentTeamBest = (
+    await admin.firestore().collection("teamBest").doc("score").get()
```
Comment (rohittanwar1) [inline functions/src/index.ts:122]: change `score` to `latest`

File: functions/src/index.ts [hunk 2]
```diff
@@ -115,30 +110,36 @@ async function teamAvg() {
...
-// export const data = functions.region(region).https.onCall(async () => {
-//   // await downloadLastData();
-//   await teamAvg();
-// });
+/**
+ * @return {String}
+ **/
+async function teamBestScore() {
+  const teamAverage = (
+    await admin.firestore().collection("TeamAvg").doc("Latest").get()
```
Comment (rohittanwar1) [inline functions/src/index.ts:118]: use lowercase `latest`.

use lowercase `teamAvg`

File: functions/src/index.ts [hunk 3]
```diff
@@ -115,30 +110,36 @@ async function teamAvg() {
     });
   }
 }
-// async function getUserData() {
```
Comment (rohittanwar1) [inline functions/src/index.ts:118]: set doc id to full timestamp for testing. later change to date only.

File: functions/src/index.ts [hunk 4]
```diff
@@ -156,5 +157,18 @@ export const backup = functions
       "NjM3ZGY0NzliZmVhYjg3ZThiZDdjMzlkLmwyT2l" +
         "5N2FRSDQySUhFVmlGUTBORnZyeGlVcVhRalZM"
     );
-    await teamAvg();
+  });
+
+export const teamAverage = functions
+  .region(region)
+  .pubsub.schedule("every 1 minutes")
```
Comment (rohittanwar1) [inline functions/src/index.ts:164]: run once a day

**Review by yogeshdave98 — APPROVED** [review]

**Pull request merged**